python/llm/src/bigdl/llm/gptq/convert/convert-gptq-to-ggml.py

```diff
@@ -0,0 +1,237 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# Convert a GPTQ quantized LLaMA model to a ggml compatible file
+# Based on: https://github.com/ggerganov/llama.cpp/blob/20a1a4e09c522a80e2a0db51643d25fa38326065/convert-gptq-to-ggml.py
+# Current supported GPTQ model: 4bits, no act-order, no safetensors.
+#
+import os
+import re
+import sys
+import json
+import struct
+import numpy as np
+import torch
+from sentencepiece import SentencePieceProcessor
+
+if len(sys.argv) != 4:
+    print("Usage: convert-gptq-to-ggml.py llamaXXb-4bit.pt tokenizer.model out.bin\n")
+    sys.exit(1)
```

Comment:
Is this for llama only?

Reply:
yes

Reply:
The we should ask the user always specify `model_family=llama` in the command line